book/src/ephemeral-local-volumes.md

```diff
@@ -38,12 +39,20 @@ Drivers must be modified (or implemented specifically) to support inline ephemer
 - Identity service
 - Node service
 
+Kubernetes 1.16 only allows using a CSI driver for an inline volume if
+its [`CSIDriverInfo`](csi-driver-object.md) object explicitly declares
+that the driver supports that kind of usage in its
+`volumeLifecycleModes` field. This is a safeguard against accidentally
+using a driver the wrong way.
```

Comment:
@msau42: this is basically the same comment as in csi-driver-object.md ("used incorrectly by Kubernetes").

Should I expand on that in ephemeral-local-volumes.md, in  csi-driver-object.md, or link to https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/20190122-csi-inline-volumes.md#support-for-inline-csi-volumes?

I'm leaning towards keeping the current text and just turning it into a link to the KEP, but I'm also fine with everything else.

Reply:
I would just link to the KEP for now.  I bring this up because there is an ask to remove this field if it's not really providing anything other than a nicer error message when someone tries to misuse a driver: https://github.com/kubernetes/kubernetes/issues/82507